src/state_machine/transitions.hpp

```diff
@@ -65,9 +65,10 @@ bool checkModulesInitialised(Logger &log, EmergencyBrakes embrakes_data, Navigat
                              Sensors sensors_data, Motors motors_data);
 
 /*
- * @brief    Returns true iff all of Embrakes, Navigation and Motors are ready.
+ * @brief    Returns true iff all modulesare ready.
```

Comment:
Typo with "modulesare" to "modules are". The meaning is clear from context but just a consistency thing.